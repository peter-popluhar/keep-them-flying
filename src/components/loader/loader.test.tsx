import { render } from "@testing-library/react";
import Loader from "./loader";

test("renders Loader", () => {
  const { container } = render(<Loader />);
  expect(container).toMatchSnapshot();
});
