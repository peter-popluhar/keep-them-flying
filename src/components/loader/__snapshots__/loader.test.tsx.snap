// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loader 1`] = `
<div>
  <div
    id="flight_loader"
  >
    <div
      class="wrapper"
    >
      <span>
        LOADING FLIGHTS
      </span>
      <div
        class="locstart"
      />
      <div
        class="flightpath"
      >
        <div
          class="airplane"
        />
      </div>
      <div
        class="locend"
      />
    </div>
  </div>
</div>
`;
